src/test/java/gift/Option/OptionRepositoryTest.java

```diff
@@ -0,0 +1,97 @@
+package gift.Option;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import gift.domain.category.Category;
+import gift.domain.category.JpaCategoryRepository;
+import gift.domain.option.JpaOptionRepository;
+import gift.domain.option.Option;
+import gift.domain.option.OptionService;
+import gift.domain.product.JpaProductRepository;
+import gift.domain.product.Product;
+import jakarta.persistence.EntityManager;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Description;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.transaction.annotation.Transactional;
+
+@SpringBootTest
```

Comment:
Repository 테스트를 위해서는 아래의 기능을 활용할 수 있습니다!
```suggestion
@DataJpaTest
```